### PR TITLE
Add filter product method in 173

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Add filter product method in 173
| Type?             | refactor
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| Sponsor company   | @PrestaShopCorp
| How to test?      | Ci is :green_apple:

### CODE REVIEW

File: src/versions/1.7.3/pages/BO/catalog/products/index.ts
Comment (Progi1984) [inline src/versions/1.7.3/pages/BO/catalog/products/index.ts:4]:
```suggestion
import {type ProductFilterMinMax} from '@data/types/product';
```

**Review by Progi1984 — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
